### PR TITLE
guile 2.0.12

### PR DESCRIPTION
**Pull request by sondr3**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Also included is the latest release of the development version of Guile.

### CODE REVIEW

File: Formula/guile.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Guile < Formula
   desc "GUILE: GNU Ubiquitous Intelligent Language for Extensions"
   homepage "https://www.gnu.org/software/guile/"
-  url "https://ftpmirror.gnu.org/guile/guile-2.0.11.tar.xz"
-  mirror "https://ftp.gnu.org/pub/gnu/guile/guile-2.0.11.tar.xz"
-  sha256 "aed0a4a6db4e310cbdfeb3613fa6f86fddc91ef624c1e3f8937a6304c69103e2"
+  url "https://ftpmirror.gnu.org/guile/guile-2.0.12.tar.xz"
+  mirror "https://ftp.gnu.org/pub/gnu/guile/guile-2.0.12.tar.xz"
+  sha256 "de8187736f9b260f2fa776ed39b52cb74dd389ccf7039c042f0606270196b7e9"
   revision 2
```
Comment (DomT4) [inline Formula/guile.rb]: Can remove this line with stable version bumps.

Comment (sondr3) [inline Formula/guile.rb]: You mean the `revision 2` line I'm assuming?

**Comment by sondr3** [comment]
Okay, removed the line and squashed the commit. I completely overlooked that line 😃

**Comment by DomT4** [comment]
Install failed on Mavericks only:

<details>
<summary>Fail</summary>
```
Making install in module
  GUILEC ice-9/eval.go
wrote `ice-9/eval.go'
  GUILEC ice-9/psyntax-pp.go
  GUILEC ice-9/boot-9.go
  GUILEC srfi/srfi-1.go
  GUILEC ice-9/vlist.go
Backtrace:
In ice-9/eval.scm:
 484: 17 [eval # #]
 481: 16 [lp (#<fluid 14>) (#<procedure 10790e310 at ice-9/eval.scm:264:7 %args>)]
In unknown file:
   ?: 15 [apply-smob/1 #<catch-closure 10799c780>]
In ice-9/eval.scm:
 484: 14 [eval # #]
 432: 13 [eval # #]
 387: 12 [eval # #]
 432: 11 [eval # #]
 387: 10 [eval # #]
 481: 9 [lp (#<fluid 24>) ("")]
 481: 8 [lp # #]
 481: 7 [lp (#<fluid 26> #<fluid 27>) (little 8)]
 481: 6 [lp (#<fluid 27>) (8)]
 481: 5 [lp (#<fluid 1>) (relative)]
 432: 4 [eval # #]
 486: 3 [eval # #]
 381: 2 [eval # #]
In unknown file:
   ?: 1 [#<variable 10780f8e0 value: #<procedure #<variable 10780f8e0 value: #<program 1072afb20>> (_ #:optional _)>> # ...]
In ice-9/eval.scm:
 481: 0 [lp (#<fluid 13>) ((#<catch-closure 10799c740>))]

ice-9/eval.scm:481:19: make[2]: *** [ice-9/psyntax-pp.go] Error 1
make[2]: *** Waiting for unfinished jobs....
Backtrace:
In ice-9/eval.scm:
 484: 17 [eval # #]
 481: 16 [lp (#<fluid 14>) (#<procedure 10f6c3070 at ice-9/eval.scm:264:7 %args>)]
In unknown file:
   ?: 15 [apply-smob/1 #<catch-closure 10f167180>]
In ice-9/eval.scm:
 484: 14 [eval # #]
 432: 13 [eval # #]
 387: 12 [eval # #]
 432: 11 [eval # #]
 387: 10 [eval # #]
 481: 9 [lp (#<fluid 24>) ("")]
 481: 8 [lp # #]
 481: 7 [lp (#<fluid 26> #<fluid 27>) (little 8)]
 481: 6 [lp (#<fluid 27>) (8)]
 481: 5 [lp (#<fluid 1>) (relative)]
 432: 4 [eval # #]
 486: 3 [eval # #]
 381: 2 [eval # #]
In unknown file:
   ?: 1 [#<variable 10efda8e0 value: #<procedure #<variable 10efda8e0 value: #<program 10ea7ab20>> (_ #:optional _)>> # ...]
In ice-9/eval.scm:
 481: 0 [lp (#<fluid 13>) ((#<catch-closure 10f167140>))]

ice-9/eval.scm:481:19: make[2]: *** [srfi/srfi-1.go] Error 1
Backtrace:
In ice-9/eval.scm:
 484: 17 [eval # #]
 481: 16 [lp (#<fluid 14>) (#<procedure 11096ccb0 at ice-9/eval.scm:264:7 %args>)]
In unknown file:
   ?: 15 [apply-smob/1 #<catch-closure 11049b1e0>]
In ice-9/eval.scm:
 484: 14 [eval # #]
 432: 13 [eval # #]
 387: 12 [eval # #]
 432: 11 [eval # #]
 387: 10 [eval # #]
 481: 9 [lp (#<fluid 24>) ("")]
 481: 8 [lp # #]
 481: 7 [lp (#<fluid 26> #<fluid 27>) (little 8)]
 481: 6 [lp (#<fluid 27>) (8)]
 481: 5 [lp (#<fluid 1>) (relative)]
 432: 4 [eval # #]
 486: 3 [eval # #]
 381: 2 [eval # #]
In unknown file:
   ?: 1 [#<variable 11028c8e0 value: #<procedure #<variable 11028c8e0 value: #<program 10fd2bb20>> (_ #:optional _)>> # ...]
In ice-9/eval.scm:
 481: 0 [lp (#<fluid 13>) ((#<catch-closure 11049b1a0>))]

ice-9/eval.scm:481:19: make[2]: *** [ice-9/vlist.go] Error 1
Backtrace:
In ice-9/eval.scm:
 484: 17 [eval # #]
 481: 16 [lp (#<fluid 14>) (#<procedure 10d4cc230 at ice-9/eval.scm:264:7 %args>)]
In unknown file:
   ?: 15 [apply-smob/1 #<catch-closure 10d0acec0>]
In ice-9/eval.scm:
 484: 14 [eval # #]
 432: 13 [eval # #]
 387: 12 [eval # #]
 432: 11 [eval # #]
 387: 10 [eval # #]
 481: 9 [lp (#<fluid 24>) ("")]
 481: 8 [lp # #]
 481: 7 [lp (#<fluid 26> #<fluid 27>) (little 8)]
 481: 6 [lp (#<fluid 27>) (8)]
 481: 5 [lp (#<fluid 1>) (relative)]
 432: 4 [eval # #]
 486: 3 [eval # #]
 381: 2 [eval # #]
In unknown file:
   ?: 1 [#<variable 10cf2a8e0 value: #<procedure #<variable 10cf2a8e0 value: #<program 10c9c9b20>> (_ #:optional _)>> # ...]
In ice-9/eval.scm:
 481: 0 [lp (#<fluid 13>) ((#<catch-closure 10d0ace40>))]

ice-9/eval.scm:481:19: make[2]: *** [ice-9/boot-9.go] Error 1
make[1]: *** [install-recursive] Error 1
make: *** [install] Error 2
```
</details>

Fine on the newer two OS X versions, so may well be an issue with Mavericks' older version of Clang by now.

**Comment by dunn** [comment]
@sondr3 would you mind asking the developers if they're willing to support Mavericks?

**Comment by sondr3** [comment]
Sure, I'll do it as soon as I get home from vacation on Sunday :)

**Comment by MikeMcQuaid** [comment]
@sondr3 Any news on that?

**Comment by ilovezfs** [comment]
Closing in favor of https://github.com/Homebrew/homebrew-core/pull/4951

**Pull request closed without merging**